.evergreen/functions.yml

```diff
@@ -698,7 +698,7 @@ functions:
           fi
 
           if [[ "$IS_RHEL" == "true" ]]; then
-            npm run --unsafe-perm --workspace @mongodb-js/compass-smoke-tests start -- --package=linux_rpm --tests=time-to-first-query
+            # npm run --unsafe-perm --workspace @mongodb-js/compass-smoke-tests start -- --package=linux_rpm --tests=time-to-first-query
```

Comment:
I had to disable this on Evergreen, since it's not running as `root` and fails: https://spruce.mongodb.com/version/67b042493a26aa0007abde21/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC&variant=smoketest-rhel-compass